v3/fleet-local/control/cfn-signal@.service

```diff
@@ -16,7 +16,7 @@ ExecStartPre=/usr/bin/systemctl is-active zk-health.service
 ExecStartPre=/usr/bin/systemctl is-active mesos-master@*
 ExecStartPre=/usr/bin/systemctl is-active marathon@*
 ExecStartPre=/usr/bin/sh -c "docker pull $($IMAGE)"
-ExecStartPre=/usr/bin/sh -c "/usr/bin/docker run --rm\
+ExecStartPre=/usr/bin/sh -c "/usr/bin/docker run --log-opt max-size=10m --log-opt max-file=10 --rm\
```

Comment:
We should make these values (`max-file`, `max-size`, etc.) configurable. Preferably, they can be set in etcd (maybe under /docker/config/logs-max-size). This will allow us to have a default, which can be overridden via the user-provided configs if needed.